tests/datasource/fluent/test_spark_datasource.py

```diff
@@ -69,6 +76,13 @@ def test_dataframe_asset(
         for asset in datasource.assets
     )
 
+    datasource_config = yaml.load(empty_data_context.fluent_config.yaml())[
+        "fluent_datasources"
+    ][datasource_name]
+    assert datasource_config["spark_config"] == {}
+    assert datasource_config["force_reuse_spark_context"] is True
+    assert datasource_config["persist"] is False
```

Comment:
This part of the test shouldn't be necessary. This is just testing the yaml serialization logic which is tested elsewhere.

Reply:
The idea was basically to ensure that the persist to False is really there. 
I will move the logic to the suggested test, but probably still keep this. Or any other suggestion?